### PR TITLE
MODORDERS-173: extract lang value from event body

### PR DESCRIPTION
**Pull request by javalkar-varun**
## Purpose

https://issues.folio.org/browse/MODORDERS-173

## Approach

`lang` is added as part of the event body so `ReceiptStatusConsistency` should extract it from the event message body

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java [hunk 1]
```diff
@@ -38,6 +38,7 @@
 public class ReceiptStatusConsistency extends AbstractHelper implements Handler<Message<JsonObject>> {
 
   private static final int LIMIT = Integer.MAX_VALUE;
+  private static final String LANG_QUERY_PARAM = "lang";
```
Comment (piotr-kalashuk) [inline src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java]: `HelperUtils.LANG` can be used instead of local constant

Comment (javalkar-varun) [reply src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java]: Done

File: src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java [hunk 2]
```diff
@@ -60,11 +62,12 @@ public void handle(Message<JsonObject> message) {
 
     String poLineIdUpdate = messageFromEventBus.getString("poLineIdUpdate");
     String query = String.format(PIECES_ENDPOINT, poLineIdUpdate, LIMIT);
-
+    
     // 1. Get all pieces for poLineId
     getPieces(query, httpClient, okapiHeaders, logger).thenAccept(piecesCollection -> {
       List<org.folio.rest.acq.model.Piece> listOfPieces = piecesCollection.getPieces();
 
+      String lang = messageFromEventBus.getString(LANG_QUERY_PARAM);
```
Comment (KVupp) [inline src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java]: minor: use the LANG directly here instead of another variable

**Comment by javalkar-varun** [comment]
lang is set [here](https://github.com/folio-org/mod-orders/blob/master/src/main/java/org/folio/rest/impl/AbstractHelper.java#L284) so it will be set whenever sendEvent is called. PiecesHelper.java, it will be set as well.

File: src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java (hunk 2 above)
Comment (javalkar-varun) [reply src/main/java/org/folio/orders/events/handlers/ReceiptStatusConsistency.java]: Done

**Pull request merged**